src/qcodes/instrument_drivers/stahl/stahl.py

```diff
@@ -144,33 +138,47 @@ class Stahl(VisaInstrument):
 
     Args:
         name
-        address: A serial port address
+        address: A serial port or TCP/IP VISA address
+        **kwargs: forwarded to base class
+
+    The TCP/IP scenario can be used when the Stahl is connected to
+    a different computer for example a Raspberry Pi running Linux and exposed
+    using something like the following script:
+
+    ```
```

Comment:
The docs build fail because this is not valid rst. You probably want to use something like https://bashtage.github.io/sphinx-material/rst-cheatsheet/rst-cheatsheet.html#blocks

Once your r&s pr merges this should run without approval allowing you to iterate faster